accounting-service/app/src/main/java/com/microservices/accounting/AccountService.java

```diff
@@ -0,0 +1,21 @@
+package com.microservices.accounting;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+/**
+ * Enabling Swagger allows to see all {@link com.microservices.accounting.controller.AccountingServiceController} endpoints
```

Comment:
Cool swagger description! Please add something similar or more detailed description to main page documentation. Used frameworks chapter.

Reply:
I created task in trello for you